source/isaaclab/isaaclab/envs/mdp/terminations.py

```diff
@@ -95,14 +105,15 @@ def joint_pos_out_of_manual_limit(
     Note:
         This function is similar to :func:`joint_pos_out_of_limit` but allows the user to specify the bounds manually.
     """
-    # extract the used quantities (to enable type-hinting)
     asset: Articulation = env.scene[asset_cfg.name]
-    if asset_cfg.joint_ids is None:
-        asset_cfg.joint_ids = slice(None)
-    # compute any violations
-    out_of_upper_limits = torch.any(asset.data.joint_pos[:, asset_cfg.joint_ids] > bounds[1], dim=1)
-    out_of_lower_limits = torch.any(asset.data.joint_pos[:, asset_cfg.joint_ids] < bounds[0], dim=1)
```

Comment:
Here the implementation is correct since it is only operating over subset of joints.